src/globus_cli/commands/endpoint/user_credential/create/s3.py

```diff
@@ -0,0 +1,50 @@
+import click
+from globus_sdk.services.gcs import UserCredentialDocument
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, endpoint_id_arg
+from globus_cli.termio import formatted_print
+
+from .._common import user_credential_create_and_update_params
+
+
+@command("s3", short_help="Create a User Credential for an S3 Storage Gateway")
+@endpoint_id_arg
+@user_credential_create_and_update_params(create=True)
+@click.argument("s3_key_id")
+@click.argument("s3_secret_key")
+@LoginManager.requires_login(LoginManager.TRANSFER_RS, LoginManager.AUTH_RS)
+def s3(
+    *,
+    login_manager: LoginManager,
+    endpoint_id,
+    storage_gateway,
+    globus_identity,
+    local_username,
+    s3_key_id,
+    s3_secret_key,
+    **kwargs,  # values from user_credential_create_and_update_params
```

Comment:
As in the other case of `**kwargs` usage, I think this is just `display_name: str | None`.